lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -1277,6 +1278,25 @@ fromCardanoWdrls = \case
             , fromCardanoLovelace coin
             )
 
+fromCardanoCerts
+    :: Cardano.TxCertificates build era
+    -> [W.RewardAccount]
+fromCardanoCerts = \case
```

Comment:
Because this function is not really just converting a "Cardano cert" to a "wallet cert", could we find a more descriptive name? Perhaps `cardanoCertKeysForWitnesses` or `cardanoCertsRequiringWitnesses`… or if you can think of something better...

Reply:
well, these not cert keys requiring wits but CANDIDATES for that. And THE KEYS when they happen to include the wallet's reward account as stake credentials. We can have in tx delegation certs belonging to other wallets. I renamed to `cardanoCertKeysForWitnesses`